test/blackbox-tests/test-cases/melange/private.t/inside/dune

```diff
@@ -3,5 +3,5 @@
  (entries c)
  (libraries app)
  (alias melange)
- (module_system es6)
- (javascript_extension mjs))
+ (module_systems
```

Comment:
I wonder if it'd make sense to add `module_system` in singular as well (in the same vein as `executables` and `executable` stanzas). If it's not very expensive to implement and maintain of course.

Reply:
I'm not so sure we should add 2 ways of doing the same thing.

Reply:
@rgrinberg It is the same with `executables` and `executable` right? I just find weird to write `module_systems` and then only pass one.

Reply:
The benefit of `executables` is that you can reuse the module group and the set of library dependencies. What would we gain from having 2 ways of doing it here?

Reply:
I think it's more similar to `foreign_stubs` or the `modules` field. I'd leave it as is.